addons/api/package.json

```diff
@@ -28,7 +28,9 @@
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-mirage": "^2.4.0",
     "ember-data": "~4.12.0",
-    "ember-fetch": "^8.1.1"
+    "ember-fetch": "^8.1.1",
+    "ember-auto-import": "^2.6.3",
+    "tracked-built-ins": "^3.1.1"
```

Comment:
I had `tracked-built-ins` in `devdependecy` first but I was getting module not found when used within the `transforms` methods, I couldn't quite figure out why it was happening, so I moved it to dependency. Let me know if we are fine with this or any other way to get this working as a `devdependecy`.

Reply:
Yeah I think it needs to be a dependency if we want to use it in API